src/solver/variable/surveyresults/surveyresults.cpp

```diff
@@ -153,7 +153,7 @@ static void ExportGridInfosAreas(const Data::Study& study,
             outThermal << cluster.minUpTime << '\t';
             outThermal << cluster.minDownTime << '\t';
             outThermal << cluster.spinning << '\t';
-            outThermal << cluster.co2 << '\t';
+            outThermal << cluster.emissions.emissionFactors[0] << '\t'; //CO2
```

Comment:
Don't use values, use the enum instead
```suggestion
            outThermal << cluster.emissions.emissionFactors[YourEnum::CO2] << '\t'; //CO2
```